src/menu.rs

```diff
@@ -10,6 +10,7 @@ pub struct Menu {
     text: CloneCell<String>,
     fg: Color,
     entries: Vec<Box<Entry>>,
+    activated: CopyCell<bool>,
```

Comment:
Because `Widget::draw()` only accepts `&self` rather than `&mut self`. I think @jackpot51 designed the API for various trait methods like this, so that only specific fields remain mutable all times while others stay immutable.